### ISSUE TITLE
Per indexer required flags setting

### ISSUE THREAD

**Issue by 2fst4u** [Prowlarr/Prowlarr#1068]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

Radarr supports required flags [as per this documentation ](https://wiki.servarr.com/radarr/settings#indexer-flags). Sonarr however does not. 

There is an open issue for Sonarr to support this, but as of currently there is no way to restrict Sonarr to only use downloads with particular flags.

### Describe the solution you'd like

If Prowlarr handled these flags before passing to the relevant app, Sonarr or Radarr, then the problem would be solved and it could be handled centrally.

### Describe alternatives you've considered

None. Waiting for the Sonarr issue to be solved would help but it doesn't make it centrally managed which would be preferred.

### Anything else?

The addition of per indexer seed requirements has helped make more of these settings centrally managed by Prowlarr which is super helpful. These indexer flags would be the cherry on top.

**Comment by bakerboy448** [comment]
Neither Radarr nor Sonarr support any Torznab flags besides freeleech.

 https://github.com/Radarr/Radarr/issues/6268

the current workaround is to create two versions of the tracker - one with freeleech only enabled and one without for the use cases.

what specific flags were you looking for?

**Comment by 2fst4u** [comment]
Freeleech, halfleech etc is exactly why I want it. But in Sonarr I can't see any way to require this. Only in Radarr.

I'm not sure I understand regarding your second question. How can I create a version of the indexer with freeleech required if there's no way to input that?

**Comment by bakerboy448** [comment]
What tracker(s)?  Pretty much all of the ones that have freeleech & support some way to search them have the ability to do freeleech only as an indexer setting.   Or if not can likely be added.

**Comment by 2fst4u** [comment]
I can't see that setting anywhere in Prowlarr for any of my indexers.

Regardless that wouldn't help. For example HD-torrents regularly does halfleech which I'm able to restrict searches and downloads to in Radarr but I can't in Sonarr.

**Comment by bakerboy448** [comment]
Again....which trackers?

**Comment by 2fst4u** [comment]
MyAnonamouse, Orpheus, AlphaRatio, IPTorrents, HDTorrents, RacingForMe don't have that setting.


Correction TVChaosUK does though I can see.

But that doesn't solve halfleech.

**Comment by bakerboy448** [comment]
- MyAnonamouse - MaM's API does not support the ability to only search FreeLeech https://github.com/Prowlarr/Prowlarr/issues/810
- Orpheus - Still needs to be moved to their API.... which may or may not support searching freeleech only https://github.com/Prowlarr/Prowlarr/issues/137
- AlphaRatio - Unsure if their site supports only searching freeleech
- IPTorrents - Has a freeleech only option 
![image](https://user-images.githubusercontent.com/55419169/179321026-2d38f364-3098-41b9-8ef3-ed69f627236f.png)
- HDTorrents - Unsure if their site supports only searching freeleech
- RacingForMe  - Unsure if their site supports only searching freeleech

So you want a dropdown/multiselect for every indexer to be able to filter post-results for freeleech/halfleech/ whatever torznab flags are determined to exist?

**Comment by 2fst4u** [comment]
Yes. I can do it in Radarr and it works fine. If I could do it in Prowlarr it would solve Sonarr's lack of support.

**Comment by qiuwei0209** [comment]
I have the same requirement. When Sonarr request Prowlarr to get torrent, If Prowlarr can control torrent with freeleech flag to respond to Sonarr. 
This will meet our needs

**Comment by quamok** [comment]
I agree that this feature would be great. I'd love to be able to use those flags from Radarr through Prowlarr:

PTP_Golden
PTP_Approved

**Comment by toqov** [comment]
I would also like to use the PTP_Golden etc tags through prowlarr. Currently I have radarr as the only instance not working through prowlarr, as the tags are not transferred and only work when directly configuring PTP in radarr.
